source/nodejs/.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+cd source/nodejs && npx lint-staged
```

Comment:
how long does it take to finish the lint check?

Reply:
it depends pretty highly on how much stuff is staged, but `eslint` runs on `source/nodejs/adaptivecards/src/*.ts` on my (not particularly beefy) machine in `3.8s`...